src/Main.res

```diff
@@ -1,6 +1,7 @@
-ReactDOM.render(
+let root = ReactDOM.Client.createRoot(ReactDOM.querySelector("#root")->Option.getExn)
+ReactDOM.Client.Root.render(
+  root,
   <React.StrictMode>
     <App />
   </React.StrictMode>,
-  ReactDOM.querySelector("#root")->Option.getExn,
 )
```

Comment:
I wonder if we could write it in this way to show case the beauty of ReScript syntax 😄 ?

```res
ReactDOM.querySelector("#root")
->Option.getExn
->ReactDOM.Client.createRoot
->ReactDOM.Client.Root.render(
  <React.StrictMode>
    <App />
  </React.StrictMode>,
)
```